### DISCUSSION TITLE
components grafana and prometheus be removed?

### DISCUSSION THREAD

**Discussion by findpather** [risingwavelabs/risingwave#10123] [Ideas]
I would like to ask where the risingwave data is stored? Can the non-essential components grafana and prometheus be removed? We don't have a browser on the server.

**Comment by hzxa21** [comment]
Grafana and prometheus are not required components for running a RisingWave cluster so you can just not create instances for grafana and prometheus. To run RisingWave without prometheus, you also need to remove the "--prometheus-listener-addr" opts when creating RisingWave nodes.

https://github.com/risingwavelabs/risingwave/blob/572780bbca08c25a5d2f71016b915733e146df04/docker/docker-compose.yml#L12
